### DISCUSSION TITLE
感觉“稍后再看”和“快速收藏”可以在所有播放界面一直显示图标，一直生效。

### DISCUSSION THREAD

**Discussion by kethoven** [the1812/Bilibili-Evolved#3138] [功能建议 / Ideas]
“稍后再看”和“快速收藏”可以在所有播放界面一直显示图标，一直生效。
就像稍后再看页面，看视频。“稍后再看”功能就不生效，如果生效的话，看过的视频，可以比较方便的移除。
在收藏夹播放页面，“快速收藏”功能就没有，如果自己想要移除收藏的话，感觉就比较合理。

**Comment by the1812** [comment]
这会出 bug 的吧, 在稍后再看界面用脚本的功能移除视频, 右边的列表还是有的; 收藏夹同理.

**Reply by kethoven** [reply]
> 这会出 bug 的吧, 在稍后再看界面用脚本的功能移除视频, 右边的列表还是有的; 收藏夹同理.

其实我觉得，列表里面不能够刷新，其实也不是问题。用网站自带的功能，移除收藏和稍后再看，列表也是不体现变化的。
我真的觉得可以尝试。只要实际上视频被移除，其实不必要在意右侧的播放列表不能显示变化的。
主要还是为了更方便吧。
